lib/content-info/index.js

```diff
@@ -0,0 +1,315 @@
+var fileType = require('file-type'),
+    mimeType = require('mime-types'),
+    mimeFormat = require('mime-format'),
+    CONTENT_TYPE = 'Content-Type',
+    CONTENT_DISPOSITION = 'Content-Disposition',
+    util = require('../util'),
+    _ = util.lodash,
+
+    /**
+     * @private
+     * @const
+     * @type {string}
+     */
+    E = '',
+
+    /**
+     * @private
+     * @const
+     * @type {string}
+     */
+    UTF8 = 'utf8',
+
+    /**
+     * @private
+     * @const
+     * @type {String}
+     */
+    doubleQuotes = '"',
+
+    /**
+     * @private
+     * @const
+     * @type {String}
+     */
+    TOKEN_$1 = '$1',
+
+    /**
+     * @private
+     * @const
+     * @type {String}
+     */
+    BINARY = 'binary',
+
+    /**
+     * @private
+     * @type {Boolean}
+     */
+    supportsBuffer = (typeof Buffer !== undefined) && _.isFunction(Buffer.byteLength),
+
+    /**
+    * @private
```

Comment:
Also need links to where this and the one below were picked from

Reply:
@kunagpal we modified the regex from the source, should we keep the link?